File: code/Python/Day3_ReverseArray.py
class Solution:
    def reverseArray(self, arr):
        
        l=len(arr)
        half_len= l//2 if (l)%2==0 else l//2+1 
        for i in range(0,half_len):
            arr[i],arr[l-i-1]=arr[l-i-1],arr[i]

File: code/Cpp/Day_3_ReverseArray.cpp
class Solution {
  public:
    void reverseArray(vector<int> &arr) {
        
        int l=arr.size();
        int half_len = l%2==0?(int)l/2:int(l)/2+1;
        
        for(int i=0;i<half_len;i++){
            int temp = arr[l-i-1];
            arr[l-i-1] = arr[i];
            arr[i]=temp;
        }
        
    }
};
